library/protocol.hpp

```diff
@@ -40,10 +40,11 @@ struct dnet_cmd_native {
 	uint64_t	size;
 };
 
-struct dnet_time_native {
-	uint64_t tsec;
-	uint64_t tnsec;
-};
+typedef dnet_time dnet_time_native;
```

Comment:
This time `dnet_cmd` and `dnet_time` used both as protocol transport structs and as core structs. It'll be good to separate core and transport from each other (because transport structs must be stable, but core structs are allowed to evolve).

While refactoring in progress, temporarily simpler to use simple type alias (not to break compilation). So, at this PR it's mistake that I declared `dnet_cmd_native` as new struct, I will fix. But later I think we should go back to this problem.